### PR TITLE
Add Requires that weren't explicit (bsc#1188646)

### PR DESCRIPTION
**Pull request by JanZerebecki**

### CODE REVIEW

File: suseconnect-ng.spec [hunk 1]
```diff
@@ -32,6 +32,26 @@ BuildRequires:  go >= 1.16
...
+Requires:       coreutils
+# ExclusiveArch from this package
+%ifarch %ix86 ia64 x86_64 %arm aarch64
+Requires:       dmidecode
+%endif
+# ExclusiveArch from this package
+%ifarch s390x
+Requires:       s390-tools
+%endif
+Requires:       systemd
```
Comment (djoreilly) [inline suseconnect-ng.spec:49]: why is systemd required?

File: suseconnect-ng.spec [hunk 2]
```diff
@@ -32,6 +32,26 @@ BuildRequires:  go >= 1.16
 Conflicts:      SUSEConnect
 Obsoletes:      SUSEConnect
 Provides:       SUSEConnect
+%if 0%{?fedora} || 0%{?rhel} || 0%{?centos_version}
```
Comment (djoreilly) [inline suseconnect-ng.spec:35]: can SUSECoonnect be installed on these OSs?

**Review by djoreilly — COMMENTED** [review]
Just a couple of questions

File: suseconnect-ng.spec (hunk 2 above)
Comment (JanZerebecki) [reply suseconnect-ng.spec:35]: Yes, in theory. Not sure if anyone did use it, maybe for an in place migration to a SUSE OS. I copied this from the ruby version.

File: suseconnect-ng.spec (hunk 1 above)
Comment (JanZerebecki) [reply suseconnect-ng.spec:49]: hwinfo.go executes systemd-detect-virt which is in that rpm.

**Review by djoreilly — APPROVED** [review]
lgtm

**Pull request merged**